test/torch_np/numpy_tests/core/test_multiarray.py

```diff
@@ -1707,7 +1711,7 @@ def test_sort_degraded(self):
         assert_equal(np.sort(d), do)
         assert_equal(d[np.argsort(d)], do)
 
-    @xpassIfTorchDynamo  # (reason="order='F'")
+    @xfail  # (reason="order='F'")
```

Comment:
why does this one fail? order="F" should break properly, no? Is it accessing `.flags` that yells at us? If so, fair enough, and we should break on that or implement them

Reply:
I plan to take a look at .flags in a follow-up.